core-bundle/src/Resources/config/services.yml

```diff
@@ -297,7 +297,7 @@ services:
             - '@contao.image.studio'
             - '@twig'
 
-    contao.insert_tag_parser:
+    contao.insert_tag.parser:
```

Comment:
I would also prefer `contao.insert_tag_parser` here. I think we could discuss whether we should strip the category from the class name or vice versa, so that we either get `contao.insert_tag.parser` or `contao.insert_tag_parser`.

Reply:
I would like it better as `contao.insert_tag.parser` as the `contao.insert_tag` namespace will most probably get more services soon.